src/pages/AddItem/styles.css

```
@@ -0,0 +1,8 @@
+.add-form {
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  width: fit-content;
+  display: flex;
+  margin: 0 auto 30px;
```

```suggestion
  display: flex;
  align-items: center;
  flex-direction: column;
  justify-content: center;
  width: fit-content;
  margin: 0 auto 30px;
```
Even though, it gives the same result, the `display:flex` is better to be declared before using any of its properties :100: